index.html

```diff
@@ -70,6 +70,219 @@ <h3><a> Let's connect </a> </h3>
         </div>
     </section>
     
+
+    <section class="portfolio-1">
+        <div class="project-1">
+            <div class="tonic-img-1">
```

Comment:
- [ ] Please wrap all cards in one section and use the grid property to align them, in your case, all sections with class portfolio-1,2... should and one main container on which you will apply that display grid property, I strongly recommend you do that.